include/seqan3/alignment/pairwise/policy/find_optimum_policy.hpp

```diff
@@ -141,6 +141,53 @@ class find_optimum_policy
             optimum = std::max(optimum, val);
         }
     }
+
+    /*!\brief Balances the total score of the alignment depending on the band settings and the alignment configuration.
```

Comment:
What does it mean to "balance a score"? Sounds like getting revenge in Mafia slang :joy: